h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
Always be refactoring. If we broke something, and we can look at it and say, "Hmm... this was never really a well thought out thing," then we should remove it rather than spend effort on restoring it to a state of still not being well thought out. If we can't make it fully baked with small effort, but we can remove it with small effort, we should do the latter. We should then prioritize it appropriately. Anything else is a disservice to ourselves and to our users who need real features, fully-baked, and may not need this feature _at all_.